cli/main.go

```diff
@@ -231,64 +224,35 @@ func main() {
 				Name:    "credentials",
 				Aliases: []string{"cr", "creds"},
 				Usage:   "Generates a proof for use with EigenPod.verifyWithdrawalCredentials()",
+				Flags: []cli.Flag{
+					BatchBySize(&batchSize, 60),
```

Comment:
Recommend defining 2 constants at the top of main rather than passing in the raw numbers